src/crypting/Main.java

```diff
@@ -0,0 +1,41 @@
+package crypting;
+
+import crypting.strategy.UnicodeCipher;
+
+public class Main {
+
+    private static CliParser parser;
+    private static Cryptor cryptor;
+
+    public static void main(String[] args) {
+        parser = new CliParser(args);
+        cryptor = new Cryptor();
+
+        if ("unicode".equals(parser.optionOf("-alg")))
+            cryptor.setCipher(new UnicodeCipher());
+
+        if (parser.optionOf("-out") != null)
+            FileReaderWriter.writeTo(parser.optionOf("-out"), getCryptedData());
```

Comment:
What this method do? what crypted data it returns?

Reply:
Fixed

Reply:
Why you closed this conversation? I don't see your answer

Reply:
> What this method do? what crypted data it returns?

Are you talking about getCryptedData() method?

Reply:
Yes

Reply:
Removed